### PR TITLE
access the focus point data through the java sdk

### PR DESCRIPTION
**Pull request by Ben-Zahler**

### CODE REVIEW

File: src/main/java/com/bynder/sdk/model/Media.java [hunk 1]
```diff
@@ -104,6 +104,9 @@
      * Generated thumbnails for the media.
      */
     private Map<String, String> thumbnails;
+
+    private Map<String, Double> activeOriginalFocusPoint;
```
[the remark this answers is not in the document]
Comment (Ben-Zahler) [reply src/main/java/com/bynder/sdk/model/Media.java]: AssetBankService.getMediaInfo returns a Media object, so I definitely need this on the Media class. Should I add it to the MediaItem as well?

File: src/main/java/com/bynder/sdk/model/Media.java [hunk 2]
```diff
@@ -105,6 +105,12 @@
      */
     private Map<String, String> thumbnails;
     /**
+     * focus point for the current media(image)
+     * map contains two key: 'x' and 'y' for the respective focus point coordinates.
+     */
+    private Map<String, Double> focusPoint;
```
Comment (danielsequeira) [inline src/main/java/com/bynder/sdk/model/Media.java]: Sorry my bad, I forgot that when you request for a specific asset, you get the activeOriginalFocusPoint in the response, I thought this just happened for the media items.

Btw you need to add @SerializedName(value = "activeOriginalFocusPoint") on top of the class member and for the javadoc just say "Focus point of the media"

File: src/main/java/com/bynder/sdk/model/Media.java [hunk 3]
```diff
@@ -198,4 +204,9 @@ public String getOriginal() {
     public List<MediaItem> getMediaItems() {
         return mediaItems;
     }
+
+    public Map<String, Double> getFocusPoint() {
+        return focusPoint;
+    }
+
```
Comment (danielsequeira) [inline src/main/java/com/bynder/sdk/model/Media.java]: remove empty line

File: src/main/java/com/bynder/sdk/model/Media.java (hunk 2 above)
Comment (Ben-Zahler) [reply src/main/java/com/bynder/sdk/model/Media.java]: just realized that, thanks

Comment (Ben-Zahler) [reply src/main/java/com/bynder/sdk/model/Media.java]: need to test it briefly

File: src/main/java/com/bynder/sdk/model/Media.java [hunk 4]
```diff
@@ -105,6 +105,13 @@
      */
     private Map<String, String> thumbnails;
     /**
+     * Focus point for the current media(image)
+     * The Map contains two keys: 'x' and 'y' for the respective focus point coordinates.
```
Comment (danielsequeira) [inline src/main/java/com/bynder/sdk/model/Media.java]: to make it coherent with the rest of the documentation, write just: "Focus point of the media."

**Pull request merged**